docs/releases.rst

```diff
@@ -20,6 +20,13 @@ calling the ``tmt-report-result`` or by calling beakerlib's
 ``rlPhaseEnd`` saved in ``results.yaml`` are now relative to the
 ``execute`` directory.
 
+The :ref:`/plugins/report/reportportal` plugin now handles the
+timestamps for ``custom`` and ``restraint`` results correctly. It
+should never happen that the result's ``start-time`` will be
```

Comment:
```suggestion
should prevent the ``start-time`` of a result being higher 
```

Reply:
Fixed in https://github.com/teemtee/tmt/pull/3356/commits/5aa14401a4ca5aa143e4b976b7719b73ef3d6e96

Thanks